dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/DataRequestCompanyIdentifierType.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandbackend.model.enums.p2p
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+/**
+ * An enum for the different identifier types that an identifier value of a stored data request can have.
+ */
+@Schema(
+    enumAsRef = true,
+)
+enum class DataRequestCompanyIdentifierType {
+    Lei,
+    Isin,
+    PermId,
+    MultipleRegexMatches,
+    DatalandCompanyId,
```

Comment:
Can the dataland companyId also be used to do data requests?

Reply:
No, this will lead to an error because companyIds are UUIDs and won't match the regexes of ISIN, PermID and LEI.

We do not want users to request data directly for a companyId in this endpoint.
They shall only pass ISIN, PErmID or LEIs.    The matching with Dataland companies is done by the community manager in the background.

Reply:
Ok, thank you.